docs/dictionary/function/processor.lcdoc

```diff
@@ -23,17 +23,15 @@ if the processor is "ARM" then ...
 
 Returns (enum):
 
--   x86: Windows systems (any architecture) and Mac OS X and Linux
-    systems with a 32 bit processor.
--   x86_64: Mac and Linux systems running a 64 bit processor
--   i386: iOS Simulator.
--   ARM: iOS and Android Devices and the Android simulator
-
+-   x86: 32 or 64 bit Windows systems and 32 bit macOS or Linux systems.
```

Comment:
Perhaps it should also be noted that the fact that 'nobody noticed' suggests that the current definition is correct - ie code using the processor worked when moved to running with 32-bit engines to 64-bit engines because it is 'doing the right thing'.

For all intents and purposes running 32-bit binaries on 64-bit OSes is essentially emulation (it's called WoW - windows on windows - for Win32) - so from the point of view of 32-bit compiled app it *is* running on a 32-bit processor / OS.